lib/modules/datasource/docker/index.ts

```diff
@@ -221,12 +220,11 @@ export class DockerDatasource extends Datasource {
   }
 
   @cache({
-    namespace: 'datasource-docker-imageconfig',
     key: (
       registryHost: string,
       dockerRepository: string,
       configDigest: string,
-    ) => `${registryHost}:${dockerRepository}@${configDigest}`,
+    ) => `imageconfig|>${registryHost}:${dockerRepository}@${configDigest}`,
```

Comment:
it's probably a bad idea to use same cache key again 🤔

Reply:
Yep, it's better to use new keys, but then we'll nuke a lot of data... Maybe it's better to refactor cache keys first

Reply:
Actually, we could leave long-term cache intact, so it's relatively safe to reset